tests/queries/0_stateless/03352_concurrent_rename_alter.sh

```diff
@@ -80,6 +83,7 @@ $CLICKHOUSE_CLIENT --query "
 # It is ok, we only check that server doesn't crash in this
 
 wait
+wait_for_mutation "t_rename_alter" "0000000001"
```

Comment:
```suggestion
```

Why do we need this? (maybe this was "fix" for PR, I've disabled this test for them here - https://github.com/ClickHouse/ClickHouse/pull/79358)

Reply:
@CurtizJ I will try to remove this